sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,490 @@
+
+        For integer/None inputs, if the estimator is a classifier and y is
+        either binary or multiclass, StratifiedKFold is used. In all other
+        cases, KFold is used.
+
+        Refer :ref:`User Guide <cross_validation>` for the various
+        cross-validation strategies that can be used here.
+
+    method_estimators : list of string or 'auto', optional
+        Methods called for each base estimator. It can be:
+
+        * if a list of string in which each string is associated to the
+          ``estimators``,
+        * if ``auto``, it will try to invoke, for each estimator,
+        ``predict_proba``, ``decision_function`` or ``predict`` in that order.
+
+    n_jobs : int, optional (default=1)
+        The number of jobs to ``fit`` the ``estimators`` in parallel. If
+        -1, then the number of jobs is set to the number of cores.
+
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by `np.random`. Used to set the ``cv``.
+
+    Attributes
+    ----------
+    estimators_ : list of estimator object
+        The base estimators fitted.
+
+    stacking_classifier_ : estimator object
+        The classifier to stacked the base estimators fitted.
+
+    method_estimators_ : list of string
+        The method used by each base estimator.
+
+    References
+    ----------
+    .. [1] Wolpert, David H. "Stacked generalization." Neural networks 5.2
+       (1992): 241-259.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> X, y = load_iris(return_X_y=True)
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.svm import LinearSVC
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from sklearn.ensemble import StackingClassifier
+    >>> estimators = [('lr', LogisticRegression()), ('svr', LinearSVC())]
+    >>> clf = StackingClassifier(estimators=estimators,
+    ...                          final_estimator=RandomForestClassifier())
+    >>> from sklearn.model_selection import train_test_split
+    >>> X_train, X_test, y_train, y_test = train_test_split(X, y)
+    >>> clf.fit(X_train, y_train).score(X_test, y_test) # doctest: +ELLIPSIS
+    0...
+
+    """
+    def __init__(self, estimators=None, final_estimator=None, cv=None,
+                 method_estimators='auto', n_jobs=1, random_state=None,
+                 verbose=0):
+        super(StackingClassifier, self).__init__(
+            estimators=estimators,
+            final_estimator=final_estimator,
+            cv=cv,
+            method_estimators=method_estimators,
+            n_jobs=n_jobs,
+            random_state=random_state,
+            verbose=verbose)
+
+    def _validate_meta_esimator(self):
```

Comment:
_validate_meta_esimator -> _validate_meta_estimator
(need to be changed here and there)